apps/project-renderer/src/components/app/index.js

```diff
@@ -50,7 +66,7 @@ const App = ( { project, page = 0, responseHash = '', startDate = 0 } ) => {
 
 		window
 			.fetch(
-				`https://api.crowdsignal.com/v4/projects/${ projectId }/responses`,
+				`https://api.crowdsignal.com/v4/projects/content/${ projectCode }`,
```

Comment:
Should, could, would... possibly maybe :) When deciding the endpoint path I was reluctant to use the same path as the protected one, mostly because `POST /projects/:ID/content` would seem to be an "author" endpoint POSTing content for the project, wouldn't it?

Now, I tried thinking what the "public" endpoint realm should be and none of the options would satisfy the actual functionality, so I went with the simplest strategy:

- `/projects[/:ID]`: protected, all derivates from it should be protected as well
- `/projects/content/:ID[/page/:pageNum]`: public, all derivates are also public

But I thought of this without consulting and I can change it really quick, just want to make sure we're OK with mixing public/protected endpoints with the same path structure. The current path mostly (only) satisfies this need of private/public "areas". Wondering if we should use `/projects` at all for the public one.

Reply:
Now the endpoints have changed as discussed:

- `GET | POST /projects/:ID/form`